js/components/contactList/contactList_controller.js

```diff
@@ -145,6 +146,15 @@ angular.module('contactsApp')
 		}
 	});
 
+	// Watch if we have an invalid contact
+	$scope.$watch('ctrl.contactList[0].fullName()', function(fullName) {
```

Comment:
if field is empty, and therefore the contact invalid, then yes, always the first one on the list.